tests/pems/crl/chain_root_valid_valid.pem

```diff
@@ -0,0 +1,95 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Not sure where to put this comment, why, if we have a script to generate these certificates, are we commiting them? I believe the reason why we wanted to have auto-gen certs during CI is to make sure that they would never expire. Since we are commiting them to the repo, when would we be running the script?

Reply:
While I would like to have our CI run this script to generate everything, unfortunately openssl 3 is required to modify the thisUpdate and nextUpdate fields of a CRL: https://github.com/aws/s2n-tls/blob/c1615a6fcf518ac9d84564d9647d97029f3106b2/tests/pems/crl/generate.sh#L235-L240

So, this would require having openssl 3 be a required dependency to run the s2n-tls unit tests.

An alternative to committing everything could be to write a small program to modify these fields ourselves, without relying on `openssl ca`. This is what I was going to do originally before I found openssl 3 added support for modifying these fields. If this option is preferred, let me know!

Reply:
I suspect that if we want to extend this script to generate all of our test certs, we will have to create a program to modify the fields ourselves, but that can be a separate PR.